Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -61,6 +66,21 @@ public void setProjectManager(IRetrieveProjectManager projectManager) {
         this.projectManager = projectManager;
     }
 
+    public String firstWords(String input, int words) {
```

Comment:
actually, this would be a perfect example of a domain object method. Put that method into your ProjectBlogEntry class. I think I would even put the code that extracts the text into that class and give the class two more methods (one to get the snippet and one to get the stripped text). And call your method "getSnippet".